### DISCUSSION TITLE
Sublime Capilot

### DISCUSSION THREAD

**Discussion by farbek1503** [community/community#64946] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

How do I install Github Capilot on **Sublime Text** ?

**Comment by arafajs** [comment, marked as answer]
GitHub Copilot **is not yet officially supported on Sublime Text**, but there is a community-maintained plugin called LSP-copilot that you can use. To install it, follow these steps:

1. Open Sublime Text.
2. Press Ctrl+Shift+P to open the command palette.
3. Type "Install Package" and press Enter.
4. In the search bar, type "lsp-copilot" and press Enter.
5. Click the "Install" button.

**Reply by farbek1503** [reply]
thank you very much
